meetings/minutes/weekly-2020-12-04.md

```diff
@@ -0,0 +1,34 @@
+# Weekly meeting
+
+**2020-12-04**
+
+Chair: @
+
+## New features
+
+* Refactor
+  * Refactored prewrite: https://github.com/tikv/tikv/pull/9114
+  * Refactored gc: https://github.com/tikv/tikv/pull/9168
+
+* Rust client
+  * Allow unbounded start key: https://github.com/tikv/client-rust/pull/202
+  * Simpler client constructor: https://github.com/tikv/client-rust/pull/201
+
+* Python client
+  * Allow to asynchronously create a client: https://github.com/tikv/client-py/pull/2
+  * Return kvpairs instead of python dict from `scan()`: https://github.com/tikv/client-py/pull/4
+
+## Test & Bugfix
+
+* Return memory locks as ScanResponse level error: https://github.com/tikv/tikv/pull/9120
+* Remove invalid lock num assertion in 1PC: https://github.com/tikv/tikv/pull/9172
+* Retry the same scan request when there is a response-level lock: https://github.com/pingcap/tidb/pull/21348
+* Disable 1PC and async commit by default: https://github.com/pingcap/tidb/pull/21438
+* Disable async commit and 1PC during the upgrade process from TiDB < 5.0: https://github.com/pingcap/tidb/pull/21365
```

Comment:
* New compatibility issue of async commit with CDC and TiFlash due to rewritting write cf records to add overlapped_rollback flag https://github.com/tikv/sig-transaction/issues/81